src/Common/tests/System/PlatformDetection.cs

```diff
@@ -23,6 +23,9 @@ public static class PlatformDetection
             File.Exists(Path.Combine(Environment.GetEnvironmentVariable("windir"), "regedit.exe")));
```

Comment:
Why I know this isn't your change I'm not sure why we chose regedit for a Nano feature query. @Priya91 any reason we chose that program to key off of?